health_check/views.py

```diff
@@ -19,4 +20,11 @@ def get(self, request, *args, **kwargs):
             errors += plugin.errors
         plugins.sort(key=lambda x: x.identifier())
         status_code = 500 if errors else 200
+
+        if 'application/json' in request.META.get('HTTP_ACCEPT', ''):
+            return JsonResponse(
+                {str(p.identifier()): str(p.pretty_status()) for p in plugins},
```

Comment:
I would put this part into a different method. That way users can overwrite the structure of the dict that gets serialised into a JSON